app/src/scan.py

```diff
@@ -458,6 +488,74 @@ def scan_domain(
 
     domain_result.dmarc.valid = len(domain_result.dmarc.errors) == 0
 
+    # TLS
```

Comment:
if you add a new section, it's a good idea to move it to a separate function, as `scan_domain` grew too long